nodes/persistence-node/src/utils/toArray.ts

```diff
@@ -0,0 +1,9 @@
+export async function toArray<T>(iterable: AsyncIterable<T>): Promise<T[]> {
```

Comment:
This isn't very descriptive, maybe 'asyncIterableToArray' would be more appropriate?